src/postgraphile/withPostGraphileContext.ts

```diff
@@ -532,15 +539,38 @@ export function debugPgClient(pgClient: PoolClient, allowExplain = false): PoolC
             const query = a && a.text ? a.text : a;
             const values = a && a.text ? a.values : b;
             if (query.match(/^\s*(select|insert|update|delete|with)\s/i) && !query.includes(';')) {
+              // Build the EXPLAIN command
+              let explainCommand = 'explain';
+              const explainOptionClauses = Object.entries(pgClient._explainOptions ?? {}).map(
+                ([option, value]) => `${option} ${value}`,
```

Comment:
`_explainOptions` isn't safe to use like this, this opens PostGraphile up to SQL injection attacks.